presets/4.3/vtx/Unify_Pro32_Nano_SA2_1.txt

```diff
@@ -22,5 +22,5 @@ vtxtable band 3 BOSCAM_E E FACTORY 5705 5685 5665 5645 5885 5905 5925 5945
 vtxtable band 4 FATSHARK F FACTORY 5740 5760 5780 5800 5820 5840 5860 5880
 vtxtable band 5 RACEBAND R FACTORY 5658 5695 5732 5769 5806 5843 5880 5917
 vtxtable powerlevels 4
-vtxtable powervalues 14 20 26 36
+vtxtable powervalues 14 20 26 28
```

Comment:
Not according to: https://www.team-blacksheep.com/vtx-table-for-betaflight.txt

Could you provide a link?

Reply:
I'd have to believe that's wrong as well. No way that tiny vtx can push 36dbm. The 28dbm I suggested works up to 630mw which seems to be spot on. 

I realized there was an issue where the video would catastrophically blackout on max power after about 2 mins of flying. 

The manual says 28dbm as well: https://www.team-blacksheep.com/media/files/tbs-unify-pro32-manual.pdf

Reply:
Thanks. Have also asked TBS. Waiting for answers.

Reply:
no way the 32 pro nano can produce more than 500ish W so id say this is correct